templates/apollo-starter-ts/model/Types.graphql

```diff
@@ -1,14 +0,0 @@
-type Note {
```

Comment:
So we might need separate issue where we going to introduce another question in init:

- Do you want to initialize project with sample schema?
yes
- What example project you want:
   x Note Example
   x Shop example (etc.)

We might have those examples as another folder in the repository and allow people to contribute their own.